src/client/utils/Intercom.tsx

```diff
@@ -47,3 +47,40 @@ export const Intercom: React.FC = () => {
 
   return null
 }
+
+export const IntercomCheckoutStyles: React.FC = () => {
+  const intercomBtnSelector = '.intercom-lightweight-app-launcher'
+  const intercomTimeout = useRef<null | NodeJS.Timeout>()
+
+  const onInteraction = () => {
+    const buttonEl: HTMLElement | null = document.querySelector(
+      intercomBtnSelector,
+    )
+    if (buttonEl !== null) {
+      if (intercomTimeout.current) {
+        clearTimeout(intercomTimeout.current)
+      }
+      buttonEl.style.opacity = '1'
+      intercomTimeout.current = setTimeout(() => {
+        buttonEl.style.opacity = '0'
+      }, 800)
+    }
+  }
+  useEffect(() => {
+    window.addEventListener('scroll', onInteraction)
+    window.addEventListener('mousemove', onInteraction)
+    // onInteraction()
```

Comment:
Yep, also the unmount. Regarding the previous comment, currently, the element is visible until the first interaction, and afterwards it gets hidden. On longer mobile screens there's no scroll, but the mousemove does trigger an interaction tick event on touchscreens so they can't click the invisible icon, it will be shown on tap